src/widgetastic_patternfly/__init__.py

```diff
@@ -2814,8 +2815,8 @@ def items(self, item_filter=None):
         stop = self.item_count + 1
         # filter via key, value pair
         if isinstance(item_filter, dict):
-            key = item_filter.keys()[0]
-            value = item_filter.values()[0]
+            key = list(item_filter.keys())[0]
```

Comment:
this should use `key, value = next(item_filter.items())`

Reply:
nifty, thanks!